plugins/objdump/README.md

```diff
@@ -0,0 +1,52 @@
+# Overview
```

Comment:
instead of `Readme.md` we put all the information into the `man` section of the plugin. Rationale - the readme is not available to a user, and is not installed. Having more than one source of information is bad.